share/module_check_a_mundo.F

```diff
@@ -2107,6 +2107,31 @@ END FUNCTION bep_bem_nbui_max
       END IF
 #endif
 
+!-----------------------------------------------------------------------
+!  The CTSM scheme may not even be compiled, so make sure it is not allowed
+!  to be run if the code is not available.
+!-----------------------------------------------------------------------
+
+#if !defined ( WRF_USE_CTSM ) 
+      oops = 0
+      DO i = 1, model_config_rec % max_dom
+         IF ( .NOT. model_config_rec % grid_allowed(i) ) CYCLE
+         IF ( model_config_rec%sf_surface_physics(i) .EQ. CTSMSCHEME ) THEN
+            oops = oops + 1
+         END IF
+      ENDDO      ! Loop over domains
+      IF ( oops .GT. 0 ) THEN
+         wrf_err_message = '--- ERROR: The CTSM surface scheme was requested in the namelist.input file.'
+         CALL wrf_debug ( 0, TRIM(wrf_err_message) )
+         wrf_err_message = '--- ERROR: However, the WRF CTSM scheme was not compiled in WRF.'
+         CALL wrf_debug ( 0, TRIM(wrf_err_message) )
+         wrf_err_message = '--- ERROR: Please set -DWRF_USE_CTSM by setting WRF_CTSM_MKFILE&
```

Comment:
Perhaps send the user to a README file? Instead of the "Please set ...", the info in the already-generated README would be better. The first couple of lines are good to keep.

Reply:
100% agreed, I don't like the message  "Please set -DWRF_USE_CTSM ...".
I changed it to "Please read doc/README.CTSM for how to compile WRF with CTSM." as you suggested.